app/scripts/views/mixins/signin-mixin.js

```diff
@@ -7,13 +7,17 @@
 define(function (require, exports, module) {
   'use strict';
 
+  const {t} = require('../base');
   const AuthErrors = require('../../lib/auth-errors');
+  const OAuthErrors = require('../../lib/oauth-errors');
   const NavigateBehavior = require('../behaviors/navigate');
   const ResumeTokenMixin = require('./resume-token-mixin');
   const VerificationMethods = require('../../lib/verification-methods');
   const VerificationReasons = require('../../lib/verification-reasons');
   const TokenCodeExperimentMixin = require('../mixins/token-code-experiment-mixin');
 
+  const TOTP_SUPPORT_URL = 'https://support.mozilla.org/en-US/kb/secure-firefox-account-two-step-authentication';
```

Comment:
I know I'm late to the party, but SUMO and MDN links should never have `en-US` in them. The site will figure out the best article for the user based on their accept-language header. Can you update this link?